scripts/calculate_aligned_extent.sql

```diff
@@ -0,0 +1,59 @@
+create or replace function calculate_aligned_extent(facility_polygon_id integer, region_id integer)
```

Comment:
We are redefining the function on every invocation of the script here. Move this to `resources/planwise/sql/` folder so the function is set up via migrations, and invoke it directly from the bash script.